lib/resources/r4/patient.yaml

```diff
@@ -0,0 +1,366 @@
+      description: The nature of the relationship between a patient and a contact person for that patient.
+      terminology:
+      - display: PatientContactRelationship
+        system: http://hl7.org/fhir/patient-contactrelationship
+        info_link: https://hl7.org/fhir/R4/valueset-patient-contactrelationship.html
+      - display: v2.0131
+        system: http://terminology.hl7.org/CodeSystem/v2-0131
+        info_link: https://hl7.org/fhir/R4/v2/0131/index.html
+      - display: Millennium Person Relationship Types
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/351
+        info_link: https://fhir.cerner.com/millennium/R4/proprietary-codes/#code-set-351-person-relationship-types
+
+- name: communication
+  required: 'No'
+  cardinality: 0..*
+  type: BackboneElement
+  description: Language which may be used to communicate with the patient about his or her health.
+  action: terminology
+  children:
+
+  - name: language
+    required: 'No'
+    cardinality: 1..1
+    type: CodeableConcept
+    description: The language used to communicate with a patient.
+    action: terminology
+    binding:
+      description: Language which may be used to communicate with the patient about his or her health.
+      terminology:
+      - display: CommonLanguages
+        system: http://hl7.org/fhir/languages
+        info_link: https://hl7.org/fhir/R4/valueset-languages.html
+      - display: IETF Identifying Languages
+        system: urn:ietf:bcp:47
+        info_link: https://tools.ietf.org/html/bcp47
+      - display: Millennium Languages
+        system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/36
+        info_link: https://fhir.cerner.com/millennium/R4/proprietary-codes/#code-set-36-languages
+
+- name: resourceType
+  required: 'Yes'
+  cardinality: 1..1
+  type: string
+  description: The type of the FHIR resource.
+  action: create
+  example: |
+    {
+      "resourceType": "Patient"
+    }
+  note: resourceType must be Patient
+  url: http://hl7.org/fhir/R4/json.html#resources
+
+- name: identifier
+  required: 'Yes'
+  cardinality: 0..*
+  type: Identifier
+  description: The identifiers for the patient.
+  action: create
+  example: |
+    {
+      "identifier": [
+        {
+          "assigner": {
+            "reference": "Organization/619848"
+          }
+        },
+        {
+          "type": {
+            "coding": [
+              {
+                "code": "MR",
+                "system": "http://hl7.org/fhir/v2/0203"
+              }
+            ]
+          },
+          "system": "urn:oid:1.1.1.1.1.1",
+          "value": "&lt;unique identifier&gt;",
+          "period": {
+            "start": "2016-01-02T00:00:00-05:00",
+            "end": "2020-01-02T00:00:00-05:00"
+          }
+        }
+      ]
+    }
+  note: Atleast one identifier should be provided, and must only contain an `assigner` field with a reference to the Organization in which the patient is being enrolled.
```

Comment:
let's use bullets to break this down, html ul and li tags should work here

same for name

Reply:
updated: 0bcdebffd7683a06e5014f464bf3715d627d7b63